.github/workflows/test-policies.yml

```diff
@@ -13,31 +13,25 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
-      - name: OPA format
-        uses: docker://openpolicyagent/opa:0.45.0
+      - name: Setup OPA
+        uses: open-policy-agent/setup-opa@v2
         with:
-          args: "fmt ./bundle --fail=true --diff"
+          version: latest
```

Comment:
Let's have pinned version here so it won't change in the future without us knowing.